src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--31">
+      <div class="calendar__day calendar__day--tuesday"></div>
```

Comment:
all modifiers should be on calendar class

```suggestion
    <div class="calendar calendar--month-length--31 calendar--start-day--tue">
      <div class="calendar__day"></div>
```

Reply:
also use mon, tue, wed, thu, fri, sat and sun values

Reply:
Добре, а чому методом як я зробила не ок?